resources/patches/ruby_241/windows_ruby_2.1_update_to_rubygems_2.4.5.1.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/lib/rubygems.rb b/lib/rubygems.rb
+index 05690c8..98e7dfa 100644
+--- a/lib/rubygems.rb
++++ b/lib/rubygems.rb
+@@ -9,7 +9,7 @@ require 'rbconfig'
+ require 'thread'
+ 
+ module Gem
+-  VERSION = '2.4.5'
++  VERSION = '2.4.5.1'
```

Comment:
Same for this one. Patch file is still being carried around, but not used. Will delete.